cross/emby/Makefile

```diff
@@ -0,0 +1,27 @@
+PKG_NAME = emby
+PKG_VERS = 3.0.5597.1
+PKG_EXT = zip
+PKG_DIST_NAME = MediaBrowser-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/MediaBrowser/MediaBrowser.Releases/raw/master/Server
+PKG_DIR = emby
+
+DEPENDS = 
+
+HOMEPAGE = http://emby.media/
+COMMENT  = Emby streams everything Your personal videos, music, photos, and live television in the palm of your hand.
+LICENSE  =
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = myInstall
+
+EXTRACT_TARGET = myExtract
+
+include ../../mk/spksrc.cross-cc.mk
+
+myInstall:
+	mkdir -p $(STAGING_INSTALL_PREFIX)/share/$(PKG_DIR)
+	tar -cf - -C $(WORK_DIR)/$(PKG_DIR) . | tar -xf - -C $(STAGING_INSTALL_PREFIX)/share/$(PKG_DIR)
+
+myExtract:
+	unzip ../../distrib/$(PKG_DIST_NAME) -d $(WORK_DIR)/$(PKG_DIR)
```

Comment:
honestly i don't remember from memory why i did this...
I'll check and if its because of the `EXTRACT_PATH` change it...

i think there was something like a folder inside the `zip` file which i'd like to exclude or something like that